e2e/nomostest/artifactregistry/helm.go

```diff
@@ -165,3 +166,10 @@ func (r *HelmChart) Push() error {
 	}
 	return nil
 }
+
+// HelmChartVersion returns the provided chart version as a function.
+func HelmChartVersion(chartVersion string) nomostest.Sha1Func {
```

Comment:
This isn't specific to artifact registry, so I don't think this function belongs here. I'm also getting rid of this package with my other PR

Reply:
/hold

Reply:
ACK. I can find a better place to put the function. If you don't mind, I would like to get this change in first to unblock 1.17.2 release, and then get #1113 merged after.

Reply:
moved to `e2e/nomostest/nt.go` to be along with other similar Sha1Funcs.